packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -0,0 +1,69 @@
+import { Link } from 'react-router-dom';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { MAX_NUM_SUGGESTIONS } from './data/constants';
+
+const SearchSuggestions = ({ autoCompleteHits, enterpriseSlug, handleViewAllClick }) => {
+  const getLinkToCourse = (course) => `/${enterpriseSlug}/course/${course.key}`;
+  const getLinkToProgram = (program) => `/${enterpriseSlug}/program/${program.aggregation_key.split(':').pop()}`;
```

Comment:
why we are using pop function here?

Reply:
aggregation key is of form 'program:uuid', we need uuid only to redirect. so its this way.
copied from [here](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/c36771b0e6c30e5ae9c161e14ac9d1be6b969c33/src/components/search/SearchProgramCard.jsx#L43)